test/files/run/wacky-value-classes.scala

```diff
@@ -0,0 +1,19 @@
+// scala/bug#9240
+final class AnyValNothing(val self: Nothing) extends AnyVal
+final class AnyValNull   (val self: Null   ) extends AnyVal
+// scala/bug#10361
```

Comment:
Your references are inverted. The below is scala/bug#9240, and the ones above are scala/bug#10361.